its/ruling/src/test/resources/expected_extended/scikit-learn/python-S1226.json

```diff
@@ -1,8 +1,8 @@
 {
-'scikit-learn:sklearn/ensemble/_forest.py':[
-2678,
-],
-'scikit-learn:sklearn/ensemble/_iforest.py':[
-236,
+"scikit-learn:sklearn/ensemble/_forest.py": [
```

Comment:
We should probably avoid reformatting files that didn't change

Reply:
Good call! Thanks

Reply:
I think reformating files is okay, as we're in a somewhat inconsistent state right now (for a long time the produce json had a special format and it's hasn't been that long since we product valid json), and I personally prefer seeing formatting-only changes rather than to see mixed formatting and functional changes when this file is actually impacted.

However, I suggest having explicit formatting-only commits when this happens so that actual ruling changes can be reviewed in a focused way.